script.js

```diff
@@ -1,4 +1,100 @@
+//global variables
+var gameMode = "rollDice"; //3 modes, get number, combine number, find winner
+var diceNumbers = []; //for each set of 2 dice rolls
+var chosenNumbersInARound = []; //for all players
+var allChosenNumbers = [];
+// var playerScore = 0;
+var player1Score = 0;
+var player2Score = 0;
+var playerNumber = 1;
+var largerNumber = 0;
+
+//generate random number 1 to 6
+function rollDice() {
+  return Math.ceil(Math.random() * 6);
+}
+
+//roll 2 numbers
+function getDiceNumbers() {
+  for (i = 0; i < 2; i += 1) {
+    diceNumbers.push(rollDice());
+  }
+  return `Hi, player ${playerNumber}, you rolled ${diceNumbers}! Which number would you choose to go first? Key in '1' or '2' and click submit!`;
+}
+
+//arrange number based on user input
+function arrangeSequence(firstChosenDice) {
+  if (Number(firstChosenDice) === 1) {
+    var combinedNumber =
+      String(diceNumbers[firstChosenDice - 1]) + String(diceNumbers[1]);
+    diceNumbers = []; //think of how not to repeat
+  } else {
+    combinedNumber =
+      String(diceNumbers[firstChosenDice - 1]) + String(diceNumbers[0]);
+    diceNumbers = [];
+  }
+  chosenNumbersInARound.push(combinedNumber);
+  allChosenNumbers.push(combinedNumber);
+  return combinedNumber;
+}
+
+function findWinner() {
```

Comment:
Same here. This function is determining the winner as well as returning a message. This generally makes your functions less reusable